src/perspective_view/panel.ts

```diff
@@ -820,10 +932,6 @@ export class PerspectivePanel extends RenderedDataPanel {
       gl.disable(WebGL2RenderingContext.BLEND);
     }
 
-    if (this.viewer.showAxisLines.value) {
```

Comment:
I'm inclined to keep the axis lines in their original (depth-correct) rendering position for now --- users could reduce the alpha value used by the max projection if they want to see them